src/lib/compress.c

```diff
@@ -1,59 +1,6 @@
-/*
- * This file is provided under a dual BSD/LGPLv2.1 license.  When using or
- * redistributing this file, you may do so under either license.
- *
- * BSD LICENSE
- *
- * tinycompress library for compress audio offload in alsa
- * Copyright (c) 2011-2012, Intel Corporation
- * All rights reserved.
- *
- * Author: Vinod Koul <vkoul@kernel.org>
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are met:
- *
- * Redistributions of source code must retain the above copyright notice,
- * this list of conditions and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice,
- * this list of conditions and the following disclaimer in the documentation
- * and/or other materials provided with the distribution.
- * Neither the name of Intel Corporation nor the names of its contributors
- * may be used to endorse or promote products derived from this software
- * without specific prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
- * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
- * THE POSSIBILITY OF SUCH DAMAGE.
- *
- * LGPL LICENSE
- *
- * tinycompress library for compress audio offload in alsa
- * Copyright (c) 2011-2012, Intel Corporation.
- *
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU Lesser General Public License,
- * version 2.1, as published by the Free Software Foundation.
- *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
- * License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to
- * the Free Software Foundation, Inc.,
- * 51 Franklin St - Fifth Floor, Boston, MA 02110-1301 USA.
- */
+/* SPDX-License-Identifier: (LGPL-2.1-only OR BSD-3-Clause) */
+/* Copyright (c) 2011-2012, Intel Corporation */
+/* Author: Vinod Koul <vkoul@kernel.org> */
```

Comment:
why not the kernel-style

// SPDX-License-Identifier: (LGPL-2.1-only OR BSD-3-Clause)
// Copyright (c) 2011-2012, Intel Corporation
//  Vinod Koul <vkoul@kernel.org

-> I don't know what the official policy is but you did this work as an Intel employee. Your email no longer reflect that fact.

Reply:
I felt that we can be simpler and use two styles, /* SPDX */ for code files.
and # SPDX for Makefiles.
Let me know if there is a strong reason to keep kernel style

The email refers to author of file and that persons updated email address, has nothing to do we who I was employed with (it was actually modified after I left Intel)

Reply:
I don't care about style, but you created your own style here. why change?